index.html

```diff
@@ -25,6 +25,17 @@
                 </a>
                 <nav id="navBar">
                     <img id="menu" src="style/img/hamburguer.svg" alt="">
+                    <div id="menuCont">
+                        <ul id="ulMenu">
+                            <li class="liMenu"> <a class="liMenuLink" href="#works">Portfolio</a></li>
+                            <li class="liMenu"> <a class="liMenuLink" href="#aboutMe">About</a></li>
+                            <div id="menuContact">
+                                <li class="liMenu1"><a href="#form">Contact</a></li>
+                                <li id="liMenuIcon"><a href="#form"> <img src="/style/img/contact.svg" alt=""></a>
+                                </li>
```

Comment:
- [ ] Brilliant submission! 💪 Great Design 🥇 But it seems this icon is not propping up on my device and that might be due to the slight omission on `.` just before `/style.......`. Kindly reference the path well so that users won't miss out on the opportunity to see all what aesthetic designs you have for them to experience.

![image](https://user-images.githubusercontent.com/99715640/220823138-dded65d9-0f5a-4f0c-b48e-8a7ddc6f9177.png)

- [ ] [OPTIONAL]_Here is a plus for this icon: It could be used with a `a` tag where it's `href` attribute is set to `mailito:example@check.ext`. Once user clicks on it, it pops up there email configs to send a mail to you. I think it broadens accessibility and smoothens user experience._

Reply:
Hi, @AbayomiOlaoye ,I didn't knew that if you put `/style....` you should add another folder "." to it, I deleted this error for each line that it was present (that were actually the ones that you were not able to see..) !! thank you for that lesson! 🤓

Also, I implemented the `mailto` to the message I con, thanks ! I appreciate your code review 😁 and I did think as well that that's a good implementation for the UI/UX !!

Reply:
@VelzckC0D3 That's good to hear 😃, and thanks for the kind words 😊.